components/more-less/more-less.js

```diff
@@ -0,0 +1,133 @@
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { LitElement, html, css } from 'lit-element/lit-element.js';
+import { LocalizeMixin } from '../localize/localize-mixin.js';
+import '../button/button-subtle.js';
+import 'd2l-icons/d2l-icon.js';
+import 'd2l-icons/tier1-icons.js';
+
+export class D2LMoreLess extends LocalizeMixin(LitElement)  {
+
+	static get properties() {
+		return {
+			expanded: { type: Boolean, reflect: true },
+			icon: { type: String, reflect: true },
```

Comment:
Are icon and text meant to be part of the public API?  I am wondering if they should really be properties?

Reply:
You're right, they shouldn't be public and potentially don't need to be properties